#! /bin/sh /usr/share/dpatch/dpatch-run
## 12_config_defaults.dpatch by Kevin Goodsell <kevin-opensource@omegacrash.net>
##
## DP: Collected adjustments to the default configuration from
## DP: Ubuntu/Medibuntu.

@DPATCH@
diff -urNad mplayer-trunk~/etc/example.conf mplayer-trunk/etc/example.conf
--- mplayer-trunk~/etc/example.conf	2010-02-04 11:33:39.000000000 -0800
+++ mplayer-trunk/etc/example.conf	2010-02-04 13:10:44.000000000 -0800
@@ -21,7 +21,7 @@
 #vc=ffmpeg12vdpau,ffwmv3vdpau,ffvc1vdpau,ffh264vdpau,ffodivxvdpau,
 
 # Specify default video driver (see -vo help for a list).
-#vo=xv
+vo=xv
 
 # Use SDL video with the aalib subdriver by default.
 #vo = sdl:aalib
@@ -75,7 +75,7 @@
 ##################
 
 # Specify default audio driver (see -ao help for a list).
-#ao=oss
+ao=pulse,alsa,
 
 # Use SDL audio driver with the esd subdriver by default.
 #ao = sdl:esd
@@ -86,6 +86,9 @@
 # Resample the sound to 44100Hz with the lavcresample audio filter.
 #af=lavcresample=44100
 
+# Specify default audio codec (see -ac help for a list).
+ac=mad,
+
 # Output audio to S/PDIF
 #ao=alsa:device=spdif
 #ac=hwac3,hwdts,hwmpa,
@@ -143,3 +146,11 @@
 
 # You can also include other configuration files.
 #include = /path/to/the/file/you/want/to/include
+
+#Screensaver support (for non gmplayer)
+stop-xscreensaver = "yes"
+
+#Disable joystick support
+#It seems to cause very odd problems on laptops
+#With accelerometers (See LP: #75925)
+nojoystick = yes
